nox/sessions.py

```diff
@@ -208,12 +208,64 @@ def _run(self, *args, env=None, **kwargs):
             kwargs.setdefault("external", "error")
 
         # If we aren't using a virtualenv allow all external programs.
-        if not isinstance(self.virtualenv, VirtualEnv):
+        if not isinstance(self.virtualenv, (CondaEnv, VirtualEnv)):
```

Comment:
flip this to check if its an instance of `ProcessEnv`?

Reply:
Both `CondaEnv` and `VirtualEnv` subclass `ProcessEnv`. Could use `type` or adjust the inheritance structure, but I added a `is_sandboxed` property, instead.